### PR TITLE
Minor update on models. READMEs fleshed out. Added Licenses.

### PR DESCRIPTION
**Pull request by NuriaQueralt**
Dear @rajaram5 @longmanplus @hbcesar could your review this PR? Thank you! NB I removed sio:has_measurement_value from models, and script.

### CODE REVIEW

File: fair-data-model/cytokine/competency-questions/CQ1.rq [hunk 1]
```diff
@@ -0,0 +1,42 @@
+# QC1 – What are all the samples (and their annotations) that were collected from a patient?
+
+PREFIX rdf:     <http://www.w3.org/1999/02/22-rdf-syntax-ns#>
```
Comment (rajaram5) [inline fair-data-model/cytokine/competency-questions/CQ1.rq:3]: This is prefix is not used in the query. Can you remove this prefix?

File: fair-data-model/cytokine/competency-questions/CQ1.rq [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+PREFIX rdfs:    <http://www.w3.org/2000/01/rdf-schema#>
```
Comment (rajaram5) [inline fair-data-model/cytokine/competency-questions/CQ1.rq]: This is prefix is not used in the query. Can you remove this prefix?

File: fair-data-model/cytokine/competency-questions/CQ1.rq [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+PREFIX xsd:     <http://www.w3.org/2001/XMLSchema#>
```
Comment (rajaram5) [inline fair-data-model/cytokine/competency-questions/CQ1.rq]: This is prefix is not used in the query. Can you remove this prefix?

File: fair-data-model/cytokine/competency-questions/CQ1.rq [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+PREFIX owl:     <http://www.w3.org/2002/07/owl#>
```
Comment (rajaram5) [inline fair-data-model/cytokine/competency-questions/CQ1.rq]: This is prefix is not used in the query. Can you remove this prefix?

File: fair-data-model/cytokine/competency-questions/CQ1.rq [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+PREFIX dct:     <http://purl.org/dc/terms/>
+PREFIX obo:     <http://purl.obolibrary.org/obo/>
+PREFIX ncit:    <http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#>
```
Comment (rajaram5) [inline fair-data-model/cytokine/competency-questions/CQ1.rq]: This is prefix is not used in the query. Can you remove this prefix?

File: fair-data-model/cytokine/competency-questions/CQ2.rq [hunk 6]
```diff
@@ -0,0 +1,54 @@
+# QC2 – What are the measurement values for the samples collected from a specific patient for a specific measurement process?
+
+PREFIX rdf:     <http://www.w3.org/1999/02/22-rdf-syntax-ns#>
```
Comment (rajaram5) [inline fair-data-model/cytokine/competency-questions/CQ2.rq:3]: This is prefix is not used in the query. Can you remove this prefix?

File: fair-data-model/cytokine/competency-questions/CQ2.rq [hunk 7]
```diff
@@ -0,0 +1,54 @@
...
+PREFIX rdfs:    <http://www.w3.org/2000/01/rdf-schema#>
```
Comment (rajaram5) [inline fair-data-model/cytokine/competency-questions/CQ2.rq]: This is prefix is not used in the query. Can you remove this prefix?

File: fair-data-model/cytokine/competency-questions/CQ2.rq [hunk 8]
```diff
@@ -0,0 +1,54 @@
...
+PREFIX xsd:     <http://www.w3.org/2001/XMLSchema#>
```
Comment (rajaram5) [inline fair-data-model/cytokine/competency-questions/CQ2.rq]: This is prefix is not used in the query. Can you remove this prefix?

File: fair-data-model/cytokine/competency-questions/CQ2.rq [hunk 9]
```diff
@@ -0,0 +1,54 @@
...
+PREFIX owl:     <http://www.w3.org/2002/07/owl#>
```
Comment (rajaram5) [inline fair-data-model/cytokine/competency-questions/CQ2.rq]: This is prefix is not used in the query. Can you remove this prefix?

File: fair-data-model/cytokine/competency-questions/CQ2.rq [hunk 10]
```diff
@@ -0,0 +1,54 @@
...
+PREFIX dct:     <http://purl.org/dc/terms/>
+PREFIX obo:     <http://purl.obolibrary.org/obo/>
+PREFIX ncit:    <http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#>
```
Comment (rajaram5) [inline fair-data-model/cytokine/competency-questions/CQ2.rq]: This is prefix is not used in the query. Can you remove this prefix?

File: fair-data-model/cytokine/competency-questions/CQ2.rq [hunk 11]
```diff
@@ -0,0 +1,54 @@
...
+PREFIX sio:     <http://semanticscience.org/resource/>
+PREFIX prov:    <http://http://http://www.w3.org/ns/prov#>
```
Comment (rajaram5) [inline fair-data-model/cytokine/competency-questions/CQ2.rq:7]: This is prefix is not used in the query. Can you remove this prefix?

File: fair-data-model/cytokine/competency-questions/CQ3.rq [hunk 12]
```diff
@@ -0,0 +1,43 @@
+# CQ3 – What are is the measurement value for a specific cytokine from a specific patient in a specific measurement process?
+
+PREFIX rdf:     <http://www.w3.org/1999/02/22-rdf-syntax-ns#>
```
Comment (rajaram5) [inline fair-data-model/cytokine/competency-questions/CQ3.rq:3]: This is prefix is not used in the query. Can you remove this prefix?

File: fair-data-model/cytokine/competency-questions/CQ3.rq [hunk 13]
```diff
@@ -0,0 +1,43 @@
...
+PREFIX rdfs:    <http://www.w3.org/2000/01/rdf-schema#>
```
Comment (rajaram5) [inline fair-data-model/cytokine/competency-questions/CQ3.rq]: This is prefix is not used in the query. Can you remove this prefix?

File: fair-data-model/cytokine/competency-questions/CQ3.rq [hunk 14]
```diff
@@ -0,0 +1,43 @@
...
+PREFIX xsd:     <http://www.w3.org/2001/XMLSchema#>
```
Comment (rajaram5) [inline fair-data-model/cytokine/competency-questions/CQ3.rq]: This is prefix is not used in the query. Can you remove this prefix?

File: fair-data-model/cytokine/competency-questions/CQ3.rq [hunk 15]
```diff
@@ -0,0 +1,43 @@
...
+PREFIX owl:     <http://www.w3.org/2002/07/owl#>
```
Comment (rajaram5) [inline fair-data-model/cytokine/competency-questions/CQ3.rq]: This is prefix is not used in the query. Can you remove this prefix?

File: fair-data-model/cytokine/competency-questions/CQ3.rq [hunk 16]
```diff
@@ -0,0 +1,43 @@
...
+                  sio:SIO_000300 ?person_study_idstring ;
+                  obo:IAO_0000219 ?person_study_role .
+
+    ?measurement_process a obo:OBI_0000070 ;
+                           sio:SIO_000291 ?trait_cytokine ;
+                           sio:SIO_000229 ?quantitative_trait ;
+                           sio:SIO_000008 ?measurement_process_date .
+
+    ?measurement_process_date a obo:NCIT_C25164 ; 
+                                dc:Date ?measurement_process_date_value .
```
Comment (rajaram5) [inline fair-data-model/cytokine/competency-questions/CQ3.rq]: Prefix `dc` is not declared. The query will throw an error

File: fair-data-model/cytokine/competency-questions/CQ4.rq [hunk 17]
```diff
@@ -0,0 +1,48 @@
+# QC4 - What are the severity score measurements of a patient and which measurement method was used?
+
+PREFIX rdf:     <http://www.w3.org/1999/02/22-rdf-syntax-ns#>
```
Comment (rajaram5) [inline fair-data-model/cytokine/competency-questions/CQ4.rq:3]: This is prefix is not used in the query. Can you remove this prefix?

File: fair-data-model/cytokine/competency-questions/CQ4.rq [hunk 18]
```diff
@@ -0,0 +1,48 @@
...
+PREFIX rdfs:    <http://www.w3.org/2000/01/rdf-schema#>
```
Comment (rajaram5) [inline fair-data-model/cytokine/competency-questions/CQ4.rq]: This is prefix is not used in the query. Can you remove this prefix?

File: fair-data-model/cytokine/competency-questions/CQ4.rq [hunk 19]
```diff
@@ -0,0 +1,48 @@
...
+PREFIX xsd:     <http://www.w3.org/2001/XMLSchema#>
```
Comment (rajaram5) [inline fair-data-model/cytokine/competency-questions/CQ4.rq]: This is prefix is not used in the query. Can you remove this prefix?

File: fair-data-model/cytokine/competency-questions/CQ4.rq [hunk 20]
```diff
@@ -0,0 +1,48 @@
...
+PREFIX owl:     <http://www.w3.org/2002/07/owl#>
```
Comment (rajaram5) [inline fair-data-model/cytokine/competency-questions/CQ4.rq]: This is prefix is not used in the query. Can you remove this prefix?

File: fair-data-model/cytokine/competency-questions/CQ4.rq [hunk 21]
```diff
@@ -0,0 +1,48 @@
...
+PREFIX dc:     <http://purl.org/dc/terms/>
+PREFIX obo:     <http://purl.obolibrary.org/obo/>
+PREFIX ncit:    <http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#>
```
Comment (rajaram5) [inline fair-data-model/cytokine/competency-questions/CQ4.rq]: This is prefix is not used in the query. Can you remove this prefix?

File: fair-data-model/cytokine/competency-questions/CQ5.rq [hunk 22]
```diff
@@ -0,0 +1,57 @@
+# QC5 - What are the cytokine measurement values and the disease severity score answers for a specific patient?
+
+PREFIX rdf:     <http://www.w3.org/1999/02/22-rdf-syntax-ns#>
```
Comment (rajaram5) [inline fair-data-model/cytokine/competency-questions/CQ5.rq:3]: This is prefix is not used in the query. Can you remove this prefix?

File: fair-data-model/cytokine/competency-questions/CQ5.rq [hunk 23]
```diff
@@ -0,0 +1,57 @@
...
+PREFIX rdfs:    <http://www.w3.org/2000/01/rdf-schema#>
```
Comment (rajaram5) [inline fair-data-model/cytokine/competency-questions/CQ5.rq]: This is prefix is not used in the query. Can you remove this prefix?

File: fair-data-model/cytokine/competency-questions/CQ5.rq [hunk 24]
```diff
@@ -0,0 +1,57 @@
...
+PREFIX xsd:     <http://www.w3.org/2001/XMLSchema#>
```
Comment (rajaram5) [inline fair-data-model/cytokine/competency-questions/CQ5.rq]: This is prefix is not used in the query. Can you remove this prefix?

File: fair-data-model/cytokine/competency-questions/CQ5.rq [hunk 25]
```diff
@@ -0,0 +1,57 @@
...
+PREFIX owl:     <http://www.w3.org/2002/07/owl#>
```
Comment (rajaram5) [inline fair-data-model/cytokine/competency-questions/CQ5.rq]: This is prefix is not used in the query. Can you remove this prefix?

File: fair-data-model/cytokine/competency-questions/CQ5.rq [hunk 26]
```diff
@@ -0,0 +1,57 @@
...
+PREFIX dc:     <http://purl.org/dc/terms/>
+PREFIX obo:     <http://purl.obolibrary.org/obo/>
+PREFIX ncit:    <http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#>
```
Comment (rajaram5) [inline fair-data-model/cytokine/competency-questions/CQ5.rq]: This is prefix is not used in the query. Can you remove this prefix?

File: fair-data-model/cytokine/competency-questions/CQ5.rq [hunk 27]
```diff
@@ -0,0 +1,57 @@
...
+
+    ?measurement_process_date a obo:NCIT_C25164 ; 
+                                dc:Date ?measurement_process_date_value . 
+
+    ?severity_score_evaluation_process a obo:NCIT_C25214 ;
+                                         sio:SIO_000229 ?severity_score_answer ;
+									     sio:SIO_000217 ?severity_score_evaluation_process_date .
+
+    ?severity_score_evaluation_process_date a obo:NCIT_C25164 ;
+                                        dct:Date ?severity_score_evaluation_process_date_value .
```
Comment (rajaram5) [inline fair-data-model/cytokine/competency-questions/CQ5.rq:38]: Prefix `dct` is not declared. The query will thrown an error

File: fair-data-model/cytokine/competency-questions/CQ6.rq [hunk 28]
```diff
@@ -0,0 +1,45 @@
+# QC6 - What are the severity score measurement and age of each patient?
+
+PREFIX rdf:     <http://www.w3.org/1999/02/22-rdf-syntax-ns#>
```
Comment (rajaram5) [inline fair-data-model/cytokine/competency-questions/CQ6.rq:3]: This is prefix is not used in the query. Can you remove this prefix?

File: fair-data-model/cytokine/competency-questions/CQ6.rq [hunk 29]
```diff
@@ -0,0 +1,45 @@
...
+PREFIX rdfs:    <http://www.w3.org/2000/01/rdf-schema#>
```
Comment (rajaram5) [inline fair-data-model/cytokine/competency-questions/CQ6.rq]: This is prefix is not used in the query. Can you remove this prefix?

File: fair-data-model/cytokine/competency-questions/CQ6.rq [hunk 30]
```diff
@@ -0,0 +1,45 @@
...
+PREFIX xsd:     <http://www.w3.org/2001/XMLSchema#>
```
Comment (rajaram5) [inline fair-data-model/cytokine/competency-questions/CQ6.rq]: This is prefix is not used in the query. Can you remove this prefix?

File: fair-data-model/cytokine/competency-questions/CQ6.rq [hunk 31]
```diff
@@ -0,0 +1,45 @@
...
+PREFIX owl:     <http://www.w3.org/2002/07/owl#>
```
Comment (rajaram5) [inline fair-data-model/cytokine/competency-questions/CQ6.rq]: This is prefix is not used in the query. Can you remove this prefix?

File: fair-data-model/cytokine/competency-questions/CQ6.rq [hunk 32]
```diff
@@ -0,0 +1,45 @@
...
+PREFIX dc:     <http://purl.org/dc/terms/>
+PREFIX obo:     <http://purl.obolibrary.org/obo/>
+PREFIX ncit:    <http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#>
```
Comment (rajaram5) [inline fair-data-model/cytokine/competency-questions/CQ6.rq]: This is prefix is not used in the query. Can you remove this prefix?

File: fair-data-model/cytokine/competency-questions/CQ6.rq [hunk 33]
```diff
@@ -0,0 +1,45 @@
...
+                  obo:IAO_0000219 ?person_patient_role ;
+                  sio:SIO_000300 ?patient_idstring .
+
+    ?severity_score_evaluation_process a obo:NCIT_C25214 ;
+                                         sio:SIO_000229 ?severity_score_answer ;
+                                         dc:conformsTo ?severity_score_measurement_method ;
+									     sio:SIO_000217 ?severity_score_evaluation_process_date .
+
+    ?severity_score_evaluation_process_date a obo:NCIT_C25164 ;
+                                        dct:Date ?severity_score_evaluation_process_date_value .
```
Comment (rajaram5) [inline fair-data-model/cytokine/competency-questions/CQ6.rq:32]: Prefix `dct` is not declared. The query will thrown an error

File: fair-data-model/cytokine/competency-questions/CQ4.rq [hunk 34]
```diff
@@ -0,0 +1,48 @@
...
+                  obo:IAO_0000219 ?person_patient_role ;
+                  sio:SIO_000300 ?patient_idstring .
+
+    ?severity_score_evaluation_process a obo:NCIT_C25214 ;
+                                         sio:SIO_000229 ?severity_score_answer ;
+                                         dc:conformsTo ?severity_score_measurement_method ;
+									     sio:SIO_000217 ?severity_score_evaluation_process_date . 
+
+    ?severity_score_evaluation_process_date a obo:NCIT_C25164 ;
+                                              dct:date  ?severity_score_evaluation_process_date_value .
```
Comment (rajaram5) [inline fair-data-model/cytokine/competency-questions/CQ4.rq:27]: Prefix `dct` is not declared. The query will throw an error.

File: fair-data-model/cytokine/competency-questions/CQ2.rq [hunk 35]
```diff
@@ -0,0 +1,54 @@
...
+    ?sampling_process_date a obo:NCIT_C25164 ;
+                      dct:date  ?sampling_process_date_value .
+
+    ?measurement_process a obo:OBI_0000070 ;
+                         prov:wasInformedBy ?sampling_process ;
+                         sio:SIO_000229 ?quantitative_trait ;
+                         sio:SIO_000008 ?measurement_process_date .
+
+    ?measurement_process_date a obo:NCIT_C25164 ; 
+                                dc:Date ?measurement_process_date_value .
```
Comment (rajaram5) [inline fair-data-model/cytokine/competency-questions/CQ2.rq]: Prefix `dc` is not declared. The query will throw an error.

**Review by rajaram5 — CHANGES_REQUESTED** [review]
There are some errors and minor issues with prefixs of SPARQL queries

File: fair-data-model/cytokine/competency-questions/CQ1.rq (hunk 1 above)
Comment (NuriaQueralt) [reply fair-data-model/cytokine/competency-questions/CQ1.rq:3]: actually it is used, it is 'a' or 'rdf:type'. should i remove it?

File: fair-data-model/cytokine/competency-questions/CQ1.rq (hunk 2 above)
Comment (NuriaQueralt) [reply fair-data-model/cytokine/competency-questions/CQ1.rq]: done

File: fair-data-model/cytokine/competency-questions/CQ1.rq (hunk 3 above)
Comment (NuriaQueralt) [reply fair-data-model/cytokine/competency-questions/CQ1.rq]: done

File: fair-data-model/cytokine/competency-questions/CQ1.rq (hunk 4 above)
Comment (NuriaQueralt) [reply fair-data-model/cytokine/competency-questions/CQ1.rq]: done

File: fair-data-model/cytokine/competency-questions/CQ1.rq (hunk 5 above)
Comment (NuriaQueralt) [reply fair-data-model/cytokine/competency-questions/CQ1.rq]: done

File: fair-data-model/cytokine/competency-questions/CQ2.rq (hunk 6 above)
Comment (NuriaQueralt) [reply fair-data-model/cytokine/competency-questions/CQ2.rq:3]: same as in CQ1

File: fair-data-model/cytokine/competency-questions/CQ2.rq (hunk 7 above)
Comment (NuriaQueralt) [reply fair-data-model/cytokine/competency-questions/CQ2.rq]: done

File: fair-data-model/cytokine/competency-questions/CQ2.rq (hunk 8 above)
Comment (NuriaQueralt) [reply fair-data-model/cytokine/competency-questions/CQ2.rq]: done

File: fair-data-model/cytokine/competency-questions/CQ2.rq (hunk 9 above)
Comment (NuriaQueralt) [reply fair-data-model/cytokine/competency-questions/CQ2.rq]: done

File: fair-data-model/cytokine/competency-questions/CQ2.rq (hunk 10 above)
Comment (NuriaQueralt) [reply fair-data-model/cytokine/competency-questions/CQ2.rq]: done

File: fair-data-model/cytokine/competency-questions/CQ2.rq (hunk 11 above)
Comment (NuriaQueralt) [reply fair-data-model/cytokine/competency-questions/CQ2.rq:7]: prov is used in line 39

File: fair-data-model/cytokine/competency-questions/CQ3.rq (hunk 12 above)
Comment (NuriaQueralt) [reply fair-data-model/cytokine/competency-questions/CQ3.rq:3]: same

File: fair-data-model/cytokine/competency-questions/CQ3.rq (hunk 13 above)
Comment (NuriaQueralt) [reply fair-data-model/cytokine/competency-questions/CQ3.rq]: done

File: fair-data-model/cytokine/competency-questions/CQ3.rq (hunk 14 above)
Comment (NuriaQueralt) [reply fair-data-model/cytokine/competency-questions/CQ3.rq]: done

File: fair-data-model/cytokine/competency-questions/CQ3.rq (hunk 15 above)
Comment (NuriaQueralt) [reply fair-data-model/cytokine/competency-questions/CQ3.rq]: done

File: fair-data-model/cytokine/competency-questions/CQ3.rq (hunk 16 above)
Comment (NuriaQueralt) [reply fair-data-model/cytokine/competency-questions/CQ3.rq]: done

File: fair-data-model/cytokine/competency-questions/CQ4.rq (hunk 17 above)
Comment (NuriaQueralt) [reply fair-data-model/cytokine/competency-questions/CQ4.rq:3]: same

File: fair-data-model/cytokine/competency-questions/CQ4.rq (hunk 18 above)
Comment (NuriaQueralt) [reply fair-data-model/cytokine/competency-questions/CQ4.rq]: done

File: fair-data-model/cytokine/competency-questions/CQ4.rq (hunk 19 above)
Comment (NuriaQueralt) [reply fair-data-model/cytokine/competency-questions/CQ4.rq]: done

File: fair-data-model/cytokine/competency-questions/CQ4.rq (hunk 20 above)
Comment (NuriaQueralt) [reply fair-data-model/cytokine/competency-questions/CQ4.rq]: done

File: fair-data-model/cytokine/competency-questions/CQ4.rq (hunk 21 above)
Comment (NuriaQueralt) [reply fair-data-model/cytokine/competency-questions/CQ4.rq]: done

File: fair-data-model/cytokine/competency-questions/CQ5.rq (hunk 22 above)
Comment (NuriaQueralt) [reply fair-data-model/cytokine/competency-questions/CQ5.rq:3]: same

File: fair-data-model/cytokine/competency-questions/CQ5.rq (hunk 23 above)
Comment (NuriaQueralt) [reply fair-data-model/cytokine/competency-questions/CQ5.rq]: done

File: fair-data-model/cytokine/competency-questions/CQ5.rq (hunk 24 above)
Comment (NuriaQueralt) [reply fair-data-model/cytokine/competency-questions/CQ5.rq]: done

File: fair-data-model/cytokine/competency-questions/CQ5.rq (hunk 25 above)
Comment (NuriaQueralt) [reply fair-data-model/cytokine/competency-questions/CQ5.rq]: done

File: fair-data-model/cytokine/competency-questions/CQ5.rq (hunk 26 above)
Comment (NuriaQueralt) [reply fair-data-model/cytokine/competency-questions/CQ5.rq]: done

File: fair-data-model/cytokine/competency-questions/CQ5.rq (hunk 27 above)
Comment (NuriaQueralt) [reply fair-data-model/cytokine/competency-questions/CQ5.rq:38]: done

File: fair-data-model/cytokine/competency-questions/CQ6.rq (hunk 28 above)
Comment (NuriaQueralt) [reply fair-data-model/cytokine/competency-questions/CQ6.rq:3]: same

File: fair-data-model/cytokine/competency-questions/CQ6.rq (hunk 29 above)
Comment (NuriaQueralt) [reply fair-data-model/cytokine/competency-questions/CQ6.rq]: done

File: fair-data-model/cytokine/competency-questions/CQ6.rq (hunk 30 above)
Comment (NuriaQueralt) [reply fair-data-model/cytokine/competency-questions/CQ6.rq]: done

File: fair-data-model/cytokine/competency-questions/CQ6.rq (hunk 31 above)
Comment (NuriaQueralt) [reply fair-data-model/cytokine/competency-questions/CQ6.rq]: done

File: fair-data-model/cytokine/competency-questions/CQ6.rq (hunk 32 above)
Comment (NuriaQueralt) [reply fair-data-model/cytokine/competency-questions/CQ6.rq]: done

File: fair-data-model/cytokine/competency-questions/CQ6.rq (hunk 33 above)
Comment (NuriaQueralt) [reply fair-data-model/cytokine/competency-questions/CQ6.rq:32]: done

File: fair-data-model/cytokine/competency-questions/CQ4.rq (hunk 34 above)
Comment (NuriaQueralt) [reply fair-data-model/cytokine/competency-questions/CQ4.rq:27]: done

File: fair-data-model/cytokine/competency-questions/CQ2.rq (hunk 35 above)
Comment (NuriaQueralt) [reply fair-data-model/cytokine/competency-questions/CQ2.rq]: done

**Comment by NuriaQueralt** [comment]
Hi @rajaram5 I resolved your requests, could you check them? tx!

File: fair-data-model/cytokine/competency-questions/CQ1.rq (hunk 1 above)
Comment (rajaram5) [reply fair-data-model/cytokine/competency-questions/CQ1.rq:3]: ah but `a` in SPARQL engines are coded so they don't require `rdf` prefix

File: fair-data-model/cytokine/competency-questions/CQ2.rq (hunk 11 above)
Comment (rajaram5) [reply fair-data-model/cytokine/competency-questions/CQ2.rq:7]: ah my bad I see it now

**Review by rajaram5 — APPROVED** [review]
Hi @NuriaQueralt As it indicated in the conversation `a` are coded in SPARQL engines so query doesn't require `rdf` prefix.

File: fair-data-model/cytokine/competency-questions/CQ1.rq (hunk 1 above)
Comment (rajaram5) [reply fair-data-model/cytokine/competency-questions/CQ1.rq:3]: sparql-a

for example see this query. I use `a` but the query works without `rdf` prefix

**Pull request merged**